### PR TITLE
Turkish (TR) Language Added

### PR DESCRIPTION
**Pull request by GlitchedSeal**

### CODE REVIEW

File: src/main/resources/localization/messages-tr.properties [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+error.invalid_time.format = Verdiğin `{0}` değerini geçerli bir süreye çeviremedim!
+error.invalid_time.min = Verdiğin ({0}) süresi minimum olan ({1}) süresinden kısaydı!
+error.invalid_time.max = Verdiğin ({0}) süresi maximum olan ({1}) süresinden uzundu!
+error.max_giveaways.guild = Zaten bu sunucuda {0} çekiliş devam ediyor!
+error.max_giveaways.channel = Zaten bu kanalda {0} çekiliş devam ediyor!
+error.invalid_winners.format = Verdiğin `{0}` değerini geçerli bir kazanan sayısına çeviremedim!
+error.invalid_winners.max = Verdiğin ({0}) kazanan sayısı 1 ile {1} arasında değildi!
+error.invalid_prize.length = Ödül uzunluğu {0} karakterden daha uzun olamaz!
+error.invalid_descr.length = Açıklama uzunluğu {0} karakterden daha uzun olamaz!
+error.invalid_emoji.length = Özel düğme yazısı {0} karakterden uzun olamaz!
```
Comment (MertBhey) [inline src/main/resources/localization/messages-tr.properties:14]: error.invalid_emoji.length = Buton içeriği {0} karakterden uzun olamaz!

**Review by MertBhey — CHANGES_REQUESTED** [review]

File: src/main/resources/localization/messages-tr.properties [hunk 2]
```diff
@@ -0,0 +1,80 @@
+error.permissions.user = Bu komutu kullanabilmek için sunucuyu yönet izininin olması gerek!
+error.permissions.perms_or_role = Bu komudu kullanabilmek için sunucuyu yönet izinin ya da {0} rolünün olması gerek!
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:2]: Same thing I wrote

[the remark this answers is not in the document]
Comment (MertBhey) [reply src/main/resources/localization/messages-tr.properties:2]: no
image

File: src/main/resources/localization/messages-tr.properties [hunk 3]
```diff
@@ -0,0 +1,80 @@
+error.permissions.user = Bu komutu kullanabilmek için sunucuyu yönet izininin olması gerek!
+error.permissions.perms_or_role = Bu komudu kullanabilmek için sunucuyu yönet izinin ya da {0} rolünün olması gerek!
+error.permissions.perms_no_role = Bu komudu kullanabilmek için sunucuyu yönet izninin olması gerek! Bir çekiliş manajeri kurulmamıştır. Bir admin bunu `{0}` ile yapabilir
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:3]: Confirmed

File: src/main/resources/localization/messages-tr.properties [hunk 4]
```diff
@@ -0,0 +1,80 @@
+error.permissions.user = Bu komutu kullanabilmek için sunucuyu yönet izininin olması gerek!
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:1]: Same thing

File: src/main/resources/localization/messages-tr.properties [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+error.giveaway_ended = Bu çekilişe katılamaz veya ayrılamazsınız çünkü bu çekiliş sona ermiştir!
+error.invalid_color = Verdiğin `{0}` değerini bir renge çeviremedim!
+error.reroll_message_option = Çekişin kazananları doğrudan orijinal mesaj üzerinden yeniden seçilebilir. (Masaüstü): Sağ Tık / (Mobil): Basılı tutma > **Uygulamalar** > **Reroll Giveaway**
+error.generic.ending = Çekilişi sonlandırmaya çalışırken bir hata oluştu.
+error.generic.delete = Çekiliş silindi, ancak çekiliş mesajını silmeye çalışırken bir hata oluştu.
+error.generic.creation = Bir çekiliş oluşturmaya çalışırken bir hata oldu.
+error.generic.enter = Çekilişe katılmaya çalışırken bir hata oluştu.
+error.generic.reroll = Çekilişin kazananlarını yeniden seçmeye çalışırken bir hata oluştu.
+error.generic = Bir hata oluştu.
+warning.no_giveaways = Herhangi bir çekiliş şu anda devam etmiyor!
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:33]: Confirmed

File: src/main/resources/localization/messages-tr.properties [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+info.about.brief = Çekilişleri hızla ve kolayca oluştur!
+info.about.long = Merhaba! Adım **GiveawayBot**, ve ben discord sunucundaki çekilişlerin hızlı ve kolay oluşturmaya yardım ederim! \
+[**jagrosh**#4824](https://jagrosh.com/) (<@113156185389092864>) tarafından yapıldım. Tüm komutlarım \
+`/g` ile başlar. Websiteme **https://giveawaybot.party/** sitesinden göz atmayı unutmayın!
+info.about.stats = Statistikler
+info.about.stats.servers = {0} sunucu
+info.about.stats.giveaways = {0} çekiliş
+info.about.links = Bağlantılar
+info.about.links.website = Website
+info.about.links.invite = Invite
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:59]: Confirmed

File: src/main/resources/localization/messages-tr.properties [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+error.generic = Bir hata oluştu.
+warning.no_giveaways = Herhangi bir çekiliş şu anda devam etmiyor!
+warning.not_supported = Bu özellik şu anda desteklenmiyor.
+warning.no_entries = Geçerli katılımcı yok, bu yüzden bir kazanan seçilemedi!
+success.giveaway_created = Çekiliş başarıyla oluşturuldu! ID: {0}
+success.entered = Çekilişe başarıyla katıldın!
+success.leave = Çekilişten başarıyla ayrıldın!
+success.ping = Pong!
+success.winner = Tebrikler {0}! **{1}** ödülünü kazandın!
+success.settings.role = {0} rölü olan kullanıcılar çekilişleri yönetebilir!
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:41]: Confirmed

File: src/main/resources/localization/messages-tr.properties [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+info.about.stats.giveaways = {0} çekiliş
+info.about.links = Bağlantılar
+info.about.links.website = Website
+info.about.links.invite = Invite
+info.about.links.support = Destek
+info.settings = ayarlar
+info.settings.owner = Sunucu Sahibi
+info.settings.premium = Premium Seviyesi
+info.settings.emoji = Çekiliş Emojisi
+info.settings.role = Manajer Rolü
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:65]: Confirmed

File: src/main/resources/localization/messages-tr.properties [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+success.giveaway_reroll = {0} kazanları yeniden seçti! Tebrikler {1}!
+success.giveaway_delete = {0} çekilişini başarıyla sildim!
+info.invite = Merhaba! Adım **GiveawayBot**! Ben çekilişlerin hızlı ve kolay oluşturmaya yardım ederim! \
+\nBeni sunucuna bu link ile davet edebilirsin:
+info.about = Tüm Hakkında
+info.about.brief = Çekilişleri hızla ve kolayca oluştur!
+info.about.long = Merhaba! Adım **GiveawayBot**, ve ben discord sunucundaki çekilişlerin hızlı ve kolay oluşturmaya yardım ederim! \
+[**jagrosh**#4824](https://jagrosh.com/) (<@113156185389092864>) tarafından yapıldım. Tüm komutlarım \
+`/g` ile başlar. Websiteme **https://giveawaybot.party/** sitesinden göz atmayı unutmayın!
+info.about.stats = Statistikler
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:54]: Confirmed

File: src/main/resources/localization/messages-tr.properties [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+warning.no_entries = Geçerli katılımcı yok, bu yüzden bir kazanan seçilemedi!
+success.giveaway_created = Çekiliş başarıyla oluşturuldu! ID: {0}
+success.entered = Çekilişe başarıyla katıldın!
+success.leave = Çekilişten başarıyla ayrıldın!
+success.ping = Pong!
+success.winner = Tebrikler {0}! **{1}** ödülünü kazandın!
+success.settings.role = {0} rölü olan kullanıcılar çekilişleri yönetebilir!
+success.settings.color = Embed rengi şimdi `{0}` rengine değiştirilmiştir.
+success.settings.emoji = Özel düğme emojisi `{0}` emojisine değiştirilmiştir.
+success.giveaway_ended = {0} çekilişini başarıyla sona erdim!
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:44]: Confirmed

File: src/main/resources/localization/messages-tr.properties (hunk 2 above)
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:2]: Confirmed

File: src/main/resources/localization/messages-tr.properties (hunk 1 above)
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:14]: Doesn't matter buton is more englishy düğme is more Turkish either way correct might change tho

File: src/main/resources/localization/messages-tr.properties [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+warning.no_giveaways = Herhangi bir çekiliş şu anda devam etmiyor!
+warning.not_supported = Bu özellik şu anda desteklenmiyor.
+warning.no_entries = Geçerli katılımcı yok, bu yüzden bir kazanan seçilemedi!
+success.giveaway_created = Çekiliş başarıyla oluşturuldu! ID: {0}
+success.entered = Çekilişe başarıyla katıldın!
+success.leave = Çekilişten başarıyla ayrıldın!
+success.ping = Pong!
+success.winner = Tebrikler {0}! **{1}** ödülünü kazandın!
+success.settings.role = {0} rölü olan kullanıcılar çekilişleri yönetebilir!
+success.settings.color = Embed rengi şimdi `{0}` rengine değiştirilmiştir.
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:42]: More simple, Confirmed

File: src/main/resources/localization/messages-tr.properties [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+success.settings.emoji = Özel düğme emojisi `{0}` emojisine değiştirilmiştir.
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:43]: More simple, Confirmed

File: src/main/resources/localization/messages-tr.properties [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+info.about.links.invite = Invite
+info.about.links.support = Destek
+info.settings = ayarlar
+info.settings.owner = Sunucu Sahibi
+info.settings.premium = Premium Seviyesi
+info.settings.emoji = Çekiliş Emojisi
+info.settings.role = Manajer Rolü
+info.settings.color = Embed Rengi
+info.settings.locale = Dil
+info.settings.etc = Hangi rollerin çekilişleri yönetebilmesini kontrol etmek için, **Sunucu Ayarları** > **Entegrasyonlar** > **GiveawayBot**
```
[the remark this answers is not in the document]
Comment (GlitchedSeal) [reply src/main/resources/localization/messages-tr.properties:68]: More simple, Confirmed

**Review by GlitchedSeal — COMMENTED** [review]
a

**Review by GlitchedSeal — COMMENTED** [review]
Fixed errors @MertBhey pointed out. Thanks!

**Pull request merged**